src/hostapi/jack/pa_jack.c

```diff
@@ -451,6 +451,49 @@ BlockingWaitEmpty( PaStream *s )
 
 /* ---- jack driver ---- */
 
+void replace_string(char *string, char *to_find, char *replacement)
+{
+    char *found_position = strstr(string, to_find);
+    while( found_position ) {
+        size_t sizeof_original = strlen(string);
+        size_t sizeof_found = strlen(to_find);
+        size_t sizeof_replacement = strlen(replacement);
+
+        size_t sizeof_destination = sizeof_original - sizeof_found + sizeof_replacement;
+        char *buffer = (char *)malloc((sizeof_destination + 1) * sizeof(char));
```

Comment:
You can get around this malloc and free, by copy the tailing part byte by byte to the new location. memmove() is your friend here.